### PR TITLE
Ability to only show day view

### PR DESCRIPTION
**Pull request by danjohnson95**
By using the `day-view-only` prop, users can now opt to just display a day view. 

* Clicking the month button at the top of the calendar has no action.
* Hovering over the month button will not change the background colour.
* Month and Year views will not be rendered.
* Added a demo on the Demo.vue
* Updated the *props* section of the readme 
* Added unit tests

### CODE REVIEW

File: src/components/Datepicker.vue [hunk 1]
```diff
@@ -40,7 +41,9 @@
         </div>
 
         <!-- Month View -->
-        <div :class="[calendarClass, 'vdp-datepicker__calendar']" v-show="showMonthView" v-bind:style="calendarStyle">
+        <div :class="[calendarClass, 'vdp-datepicker__calendar']" v-show="showMonthView"
+             v-if="!dayViewOnly"
```
Comment (charliekassel) [inline src/components/Datepicker.vue]: I don't think `v-show` and `v-if`should be used together on the same element

File: src/components/Datepicker.vue [hunk 2]
```diff
@@ -60,7 +63,9 @@
         </div>
 
         <!-- Year View -->
-        <div :class="[calendarClass, 'vdp-datepicker__calendar']" v-show="showYearView" v-bind:style="calendarStyle">
+        <div :class="[calendarClass, 'vdp-datepicker__calendar']" v-show="showYearView"
+             v-if="!dayViewOnly"
```
Comment (charliekassel) [inline src/components/Datepicker.vue]: Ditto. Perhaps have this directive on a `template` wrapper

Comment (danjohnson95) [inline src/components/Datepicker.vue]: I hear you. I've just updated the PR with your suggested changes

**Comment by charliekassel** [comment]
Thanks @danjohnson95

**Pull request merged**